Plinic/Back-End/PostAPI/PostAPI.swift

```diff
@@ -44,4 +47,42 @@ final class PostAPI: ObservableObject {
             }
         }
     }
+    
+    // MARK: - 게시물 상세(GET)
+    func getPostDetail(postID: Int?, _ completion: @escaping ((Result<PostDetail, Error>) -> Void)) {
+        // FIXME: - Error : 내가 하고 싶은 것: postDetailURL에 postID를 집어넣어서 requestUrl를 만들어주고 그 값을 받아온다.
+        
+        guard let postID = postID else {
+            return
+        }
```

Comment:
```suggestion
    func getPostDetail(postID: Int, _ completion: @escaping ((Result<PostDetail, Error>) -> Void)) {
```
getPost API 는 id 값이 필수인데 굳이 optional 로 할 필요가 없을것같음.
주석도 더이상 필요하지 않을듯

Reply:
맞는 말인 것 같습니다 바로 적용하겠습니다!!